docs/setup/docker.asciidoc

```diff
@@ -39,6 +39,12 @@ Alternatively, you can download other Docker images that contain only features
 available under the Apache 2.0 license. To download the images, go to 
 https://www.docker.elastic.co[www.docker.elastic.co].
 
```

Comment:
```suggestion

[float]
```
Can we add a float here to avoid creating a new page break?

Reply:
Forget it.  It's too much trouble to suggest improvements to the Elastic site.  This has been going on for over a month

Reply:
Sorry, you felt there was too much friction with the process. The suggestion he was valid in that there was an unnecessary page break. All that was needed here is to click the "Commit suggestion" button. @mistic will take over and get this change merged in.

Reply:
Sorry.  I misunderstood the request.  I thought I was being asked to submit a new commit with the line removed rather than approve a change that had already been made.